### ISSUE TITLE
missing when online

### ISSUE THREAD

**Issue by charliegc4** [bugfroggy/AnimatedCursor#4]
Installed the mod, but when I am connected to the Internet it does not display. I can configure it, but it does not display.

If I disconnect from the Internet before launching Minecraft, it works just fine.

**Comment by charliegc4** [comment]
A month and no reply, not even an acknowledgement?

Anyone have a solution for this?

**Comment by robere2** [comment]
Are you using any other mods by any chance?

**Comment by charliegc4** [comment]
Yes, a whole bunch of them; Minecraft reports 79 in fact, though some of those are libraries and children. I actually tried loading with just this mod, but I get the same thing. 

Anyway, I don't play online, so I just have to make sure I block Java in my firewall, then everything works just fine.
